components/textFields/AdvancedOpenTextField.js

```diff
@@ -61,10 +75,11 @@ const AdvancedOpenTextField = (props) => {
             setButtonAppearance('outline');
         }
     }
-    if(value.length > data.maxLength && !isInvalid) {
-        setIsInvalid(true);
-    } else if(isInvalid && value.length <= data.maxLength) {
-        setIsInvalid(false);
+
+    if(value.length > data.maxLength && !invalidLength) {
```

Comment:
it seems like when i console log data.maxLength, it is always undefined so the length check doesn't seem to do anything?

Reply:
the length check is there in case a question sets that value (although I don't think any do), and was created last year. Should we just remove it or leave that in case we want it in the future?

Reply:
i guess we can leave it and add some comments documenting the current status for the future :)